### PR TITLE
Code Review: Add XCode 13.4 build support  (#44096)

### PR DESCRIPTION
**Pull request by sketch-joseluis**
Connect sketch-hq/sketch#44096

Created by `bo pr`.

### CODE REVIEW

**Review by SheffieldKevin — APPROVED** [review]

**Review by whakkee — APPROVED** [review]
re-approved as there were changes after Kevin's approval

**Pull request merged**
